Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -1829,6 +1829,13 @@ void WebsiteDataStore::setStorageSiteValidationEnabled(bool enabled)
         networkProcess->send(Messages::NetworkProcess::SetStorageSiteValidationEnabled(sessionID(), m_storageSiteValidationEnabled), 0);
 }
 
+void WebsiteDataStore::setPersistedSites(HashSet<Site>&& persistedSites)
+{
+    m_persistedSites = WTFMove(persistedSites);
+    if (RefPtr networkProcess = networkProcessIfExists())
```

Comment:
In the case where the network process isn't launched yet, we need to pass the persisted domains to the network process during launch (via the `WebsiteDataStoreParameters` probably). This is also important in case the network process crashes and relaunches.

Reply:
Nice catch; added to `ResourceLoadStatisticsParameters`.